pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+requires = ["setuptools>=60.2.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "torch_brain"
+version = "0.1.0"
+description = "A package for deep learning models for neuroscience"
+readme = "README.md"
+authors = [
+    {name = "Mehdi Azabou", email = "mehdiazabou@gmail.com"},
+    {name = "Vinam Arora", email = "vinam.arora@gmail.com"},
+]
+license = {text = "MIT"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License", 
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "temporaldata",
+    "numpy<2.0.0",
+    "torch>=2.0.0",
```

Comment:
Can we make these:
```
numpy~=1.0
torch~=2.0
```
That indicates to the package managers that we're okay with any `numpy` and `torch` with major-versions 1 and 2 respectively.